src/@commonsku/styles/tables/VirtualTable.tsx

```diff
@@ -62,6 +62,7 @@ export type VirtualTableProps<
   rowGroupStyles?: (value: {row: Row<RowType>, style: React.CSSProperties }) => React.CSSProperties;
   rowStyles?: (value: {row: Row<RowType>, style: React.CSSProperties }) => React.CSSProperties;
   gutterSize?: number;
+  onSortChange?: (sortState: UseSortByState<RowType>) => void;
```

Comment:
I don't think that any code is using the `onSort` prop.  Can you get rid of it?  I think it's confusing to have both `onSortChange` and `onSort` with slightly different semantics.